lib/config/rpcProviderProdConfig.json

```diff
@@ -3,30 +3,35 @@
     "chainId": 42220,
     "useMultiProviderProb": 1,
     "providerInitialWeights": [1, 0],
-    "providerUrls": ["QUICKNODE_42220", "INFURA_42220"]
+    "providerUrls": ["QUICKNODE_42220", "INFURA_42220"],
+    "enableDbSync": true
```

Comment:
Shouldn't `enableDbSync` be false..?

Reply:
Sorry this PR is to only support disabling DB sync. We have other PRs to actually disable DB sync for certain chains.

Reply:
PR title is updated.